File: src/main/java/it/contrader/model/BiomedicalData.java
package it.contrader.model;

import java.util.List;

import javax.persistence.*;

import javax.validation.constraints.NotNull;

import org.springframework.lang.Nullable;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class BiomedicalData {

	@Id
	@Column(name = "idBiomedicalData")
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Integer idBiomedicalData;

	@Column(name = "hearthbeat")
	private int hearthbeat;

	@Column(name = "bloodPressure")
	private int bloodPressure;
	
	@Column(name = "height")
	private float height;
	
	@Column(name = "weight")
	private float weight;
	
	@Column(name = "fatMass")
	private float fatMass;
	
	@Column(name = "fatFreeMass")
	private float fatFreeMass;
	
	@Column(name = "deleted")
	private int deleted;
	

}

File: src/app/models/BiomedicalData.ts
export class BiomedicalData {
    id: number;
    hearthbeat: number;
    bloodPressure: number;
    height: number;
    weight: number;
    fatMass: number;
    fatFreeMass: number;
    deleted: number;




    // tslint:disable-next-line:max-line-length
    constructor(id: number, hearthbeat: number, bloodPressure: number,height: number,weight: number,fatMass: number,fatFreeMass: number,deleted: number) {
        this.id = id;
        this.hearthbeat = hearthbeat;
        this.bloodPressure = bloodPressure;
        this.height = height;
        this.weight= weight;
        this.fatMass = fatMass;
        this.fatFreeMass=fatFreeMass;
        this.deleted=deleted;


    }

}